project-memberships-permissions

From: Bryan Larsen <bryan@larsen.st>

Then permissions -- only the project owner (and admins) can manipulate these project memberships:
---
 app/models/project_membership.rb |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/app/models/project_membership.rb b/app/models/project_membership.rb
index 65f0b8b..143d1b7 100644
--- a/app/models/project_membership.rb
+++ b/app/models/project_membership.rb
@@ -13,15 +13,15 @@ class ProjectMembership < ActiveRecord::Base
   # --- Permissions --- #
 
   def create_permitted?
-    acting_user.administrator?
+    acting_user.administrator? || project.owner_is?(acting_user)
   end
 
   def update_permitted?
-    acting_user.administrator?
+    acting_user.administrator? || project.owner_is?(acting_user)
   end
 
   def destroy_permitted?
-    acting_user.administrator?
+    acting_user.administrator? || project.owner_is?(acting_user)
   end
 
   def view_permitted?(field)
